codereview.js

```
@@ -1,38 +1,57 @@
+// Unused variable (AI should flag this as an issue)
+var unusedVariable = 'This is never used';
 
-// Function calls
+// Poorly formatted function (AI should suggest fixing spacing)
+function addNumbers(a, b) {
+	return a + b;
+}
+
+// Security vulnerability (AI should suggest avoiding eval)
+function executeCode(code) {
+	return eval(code); // Potential security risk
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)